app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
When mentor checking mine task, he told me, that "All friends should be vaccinated" - is not specific enough phrase, so change it into a more specific, like: f"{friend[name]} shoud vaccinate" or f"friend[name]} vaccinate is expired" or smth. So change it.